build.gradle

```diff
@@ -5,6 +5,7 @@ buildscript {
   dependencies {
     classpath 'com.android.tools.build:gradle:2.2.2'
     classpath 'com.github.ben-manes:gradle-versions-plugin:0.13.0'
+    classpath 'com.neenbedankt.gradle.plugins:android-apt:1.8'
```

Comment:
You don't need this anymore with 2.2.0 as of gadle build tools. Use 'annotationProcessor' instead